src/components/create/create.component.js

```diff
@@ -3,26 +3,24 @@ import PropTypes from 'prop-types';
 import tagComponent from '../../utils/helpers/tags/tags';
 import CreateStyle from './create.style';
 
-class Create extends React.Component {
+const Create = (props) => {
   /** Returns the props for the component. */
-  linkProps = () => {
-    const { ...props } = this.props;
+  const linkProps = () => {
```

Comment:
Sorry, looking at this again, does this even need to be a function? Looks like something like this could work:
```
const linkProps = {
  iconAlign: 'right',
  icon: 'add',
  ...props
}
```

Reply:
You're right, updated! :)